src/effects/Effect.h

```diff
@@ -180,11 +180,13 @@ class AUDACITY_DLL_API Effect /* not final */
 
    static void IncEffectCounter(){ nEffectsDone++;}
 
-   bool EnableApply(bool enable = true);
+   static
+   bool EnableApply(wxWindow *parent, bool enable = true);
```

Comment:
As a members those functions make sense here, but not much sense as a statics. I'd suggest to move them out from these class completely

Reply:
I don't know why it's so bad to make them statics, but as functions related specially to user interface, they might make more sense as static utility members of EffectUIValidator.